beacon-chain/blockchain/receive_block.go

```diff
@@ -40,7 +40,7 @@ func (s *Service) ReceiveBlock(ctx context.Context, block *ethpb.SignedBeaconBlo
 
 	// Update and save head block after fork choice.
 	if err := s.updateHead(ctx, s.getJustifiedBalances()); err != nil {
-		return errors.Wrap(err, "could not update head")
+		log.Warnf("Resolving fork due to new attestation: %v", err)
```

Comment:
Fork choice failing should not interrupt the whole receive block routine

```suggestion
		log.Warnf("Could not update head: %v", err)
```